### ISSUE TITLE
404 on setting up the GCE example.

### ISSUE THREAD

**Issue by jackzampolin** [jetstack/kube-lego#39]
So I had this running with the default setup. It fetched the certs and was able to see the site up and running! Then I wanted to change some names and reorganize the files so I tore down the cluster and rebuilt it. I haven't been able to get it running a second time! I've gone ahead and [created a gist with the appropriate files](https://gist.github.com/jackzampolin/4ab03364ecf8e628103e60d83ae9d868).

I tried some of the troubleshooting steps from #15 but was again unable to get the certs to authenticate.

Any help would be greatly appreciated @simonswine! 

<img width="1165" alt="screenshot 2016-09-30 17 16 40" src="https://cloud.githubusercontent.com/assets/7452680/19010288/a9ab7944-8732-11e6-95af-d55259c25d46.png">

**Comment by simonswine** [comment]
@jackzampolin thanks for providing the yaml files, helps a lot. Everything looks fine, as even GCE recognises all the services as healthy:

`{"k8s-be-31019--abb61364079302df":"HEALTHY","k8s-be-31137--abb61364079302df":"HEALTHY","k8s-be-31751--abb61364079302df":"HEALTHY"}` 

Can you test that for me:
* Is `jackserver` reachable as expected under http://jackzampolin.com/?
* Output of `kubectl describe --namespace prodenv svc/kube-lego-gce`

**Comment by jackzampolin** [comment]
@simonswine Can't reach the server at `http://*.jackzampolin.com`. It auto redirects to `https://`. 

```
Name:			kube-lego-gce
Namespace:		prodenv
Labels:			<none>
Selector:		<none>
Type:			NodePort
IP:			10.51.255.213
Port:			<unset>	8080/TCP
NodePort:		<unset>	31751/TCP
Endpoints:		10.48.1.6:8080
Session Affinity:	None
No events.
```

I'm now getting `Too many certificates already issued for exact set of domains: jackzampolin.com,www.jackzampolin.com jackzampolin.com:acme` which would indicate I have to wait for a week ☹️ and I have done everything right. I'll go ahead and close this unless the above output indicates something is amiss.

**Comment by johnparn** [comment]
@jackzampolin What did you do to make it work? 

I have a similar issue, see #44. I got it working somewhat, then I made some minor changes and now I can only get one site to respond and without TLS.

**Comment by jackzampolin** [comment]
@johnparn I still haven't been able to make it work. SSL is more of a nice to have for my project (And I've used up my cert reqs this week 🙊) so I've put it on the back burner. I'm going to try the nginx example this week and see if the results are different.

**Comment by johnparn** [comment]
@jackzampolin Thanks! I'm eager to hear if nginx will work any better. 

I've tried different ways of doing it. I have two sites that both needs SSL/TLS and I can get one site up running with a cert, however the second site I start will always return a _default backend - 404_. I've tried using the same namespace for all sites including kube-lego, separate namespace for all and putting kube-lego in the same namespace as each service. Still just one site will work.

Here are the files that I've used. https://gist.github.com/johnparn/ce0e025e8c015de812c0b84ef8b1faf9
